packages/e2e-tests/plugins/custom-post-types.php

```diff
@@ -39,7 +39,7 @@ function public_queryable_true_public_false_cpt() {
 			'show_in_rest'       => true,
 			'public'             => false,
 			'publicly_queryable' => true,
-			'supports'           => array( 'title', 'editor', 'revisions' ),
+			'supports'           => array( 'excerpt', 'title', 'editor', 'revisions' ),
```

Comment:
The change in this PR highlighted a bug in Core that I tracked here https://core.trac.wordpress.org/ticket/53813
The change here just avoids making this bug fail the e2e test that is intended for something completely different.